tcms/rpc/api/testexecution.py

```diff
@@ -185,9 +185,11 @@ def update(execution_id, values, **kwargs):
     if values.get("status"):
         now = timezone.now()
         if test_execution.status.weight != 0:
-            test_execution.close_date = now
+            test_execution.stop_date = now
+            test_execution.start_date = now
```

Comment:
Don't modify `start_date` here nor below!

I'm not sure if we need to do this ATM.